### ISSUE TITLE
Add support for Moonbeam

### ISSUE THREAD

**Issue by Spenhouet** [rotki/rotki#3989]
Please support the Moonbeam chain and holdings on that chain.

Here the blockchain explorer: https://blockscout.moonbeam.network/

**Comment by gilescope** [comment]
Yeah that would be nice. In particular calculation of staking as that's a lot of small transactions.
